cache/store.go

```diff
@@ -45,6 +45,8 @@ type Store[T any] interface {
 }
 
 // Expirable is an interface for a cache store that supports expiration.
+// It extends the Store interface.
+// It also provides disk persistence.
```

Comment:
I started with a `persistable` interface. Now, LRU does not need persistence in my opinion because items can be evicted at every moment, there is no way to know what is being persisted. I don't see any value there. Also this is a cache so in most cases we don't need to persist it. For the expiration cache, users can configure the retention period, so there can be use cases where we want to start with a snapshot. That's why I finally decided to add the `persist` method in the `Expirable` interface.

But I guess there might be other implementations in the future that may need it. I'll reintroduce `persistable`.